### PR TITLE
add @react-native-community/datetimepicker specific Props for DateTimePicker

### PR DESCRIPTION
**Pull request by Macrow**
add @react-native-community/datetimepicker specific Props for DateTimePicker

### CODE REVIEW

File: src/components/dateTimePicker/index.js [hunk 1]
```diff
@@ -167,7 +183,7 @@ class DateTimePicker extends BaseComponent {
 
   renderDateTimePicker() {
     const {chosenDate, showExpandableOverlay} = this.state;
-    const {mode, minimumDate, maximumDate} = this.props;
+    const {mode, minimumDate, maximumDate, locale, is24Hour, minuteInterval, timeZoneOffsetInMinutes} = this.props;
```
Comment (ethanshar) [inline src/components/dateTimePicker/index.js]: I guess at this point we can pass on the other props as `...other`
```suggestion
    const {mode, minimumDate, maximumDate, ...others} = this.props;
```

File: src/components/dateTimePicker/index.js [hunk 2]
```diff
@@ -177,6 +193,10 @@ class DateTimePicker extends BaseComponent {
           onChange={this.handleChange}
           minimumDate={minimumDate}
           maximumDate={maximumDate}
+          locale={locale}
```
Comment (ethanshar) [inline src/components/dateTimePicker/index.js]: pass `{...others}` instead

File: src/components/dateTimePicker/index.js [hunk 3]
```diff
@@ -61,6 +61,22 @@ class DateTimePicker extends BaseComponent {
...
+     */
+    is24Hour: PropTypes.bool,
+    /**
+     * The interval at which minutes can be selected. Possible values are: 1, 2, 3, 4, 5, 6, 10, 12, 15, 20, 30 (iOS only)
+     */
+    minuteInterval: PropTypes.number,
+    /**
+     * Allows changing of the timeZone of the date picker. By default it uses the device's time zone (iOS only)
+     */
+    timeZoneOffsetInMinutes: PropTypes.number,
```
Comment (ethanshar) [inline src/components/dateTimePicker/index.js]: spread the whole `RNDateTimePicker` prop types 
`...RNDateTimePicker.propTypes` should do the trick

File: src/components/dateTimePicker/index.js (hunk 2 above)
Comment (Macrow) [reply src/components/dateTimePicker/index.js]: It can't be better

**Comment by ethanshar** [comment]
@Macrow 
Your original PR was actually fine, I merged and fix that last change.

**Pull request merged**